core/src/operations/mod.rs

```diff
@@ -131,6 +131,10 @@ pub enum Operation {
     /// operation is undefined. If the third element is greater than 1, execution fails.
     U32addc,
 
+    /// Pops three elements off the stack, adds them together, and splits the result into upper
+    /// and lower 32-bit values. Then pushes the result back onto the stack.
+    U32add3,
+
```

Comment:
I'd probably get rid of `U32addc` variant. I don't see a strong reason to keep them both, even temporarily.

Reply:
Done